### PR TITLE
[13.0][FIX] hr_employee_firstname: adapt fields access rights to V13

### PR DESCRIPTION
**Pull request by astirpe**
Fixes following error:

```
  File "/something/odoo-13.0/addons/hr/models/hr_employee.py", line 133, in _read
    res = self.env['hr.employee.public'].browse(self.ids).read(fields)
  File "/something/odoo-13.0/odoo/models.py", line 2896, in read
    raise ValueError("Invalid field %r on model %r" % (name, self._name))
ValueError: Invalid field 'firstname' on model 'hr.employee.public'
```

### CODE REVIEW

**Review by albariera — APPROVED** [review]

**Review by Saran440 — COMMENTED** [review]
LGTM :+1: 
In module v14 is not have file `hr_employee_public.py`
Do you fix file hr_employee_public.py to V14 ?
if yes, please split commit for cherry-pick it to v14.

**Comment by astirpe** [comment]
@Saran440 I didn't fix the V14 yet. Anyway it seems to me that the fix on `hr_employee_public.py` is also needed for V14

File: hr_employee_firstname/models/hr_employee_public.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    # Fields coming from hr.employee.base
+    firstname = fields.Char(readonly=True)
```
Comment (AmmarOfficewalaSerpentCS) [inline hr_employee_firstname/models/hr_employee_public.py:11]: @astirpe ,

If you added a field on an hr.employee.base then no need to add on a public 
Please remove it

**Review by AmmarOfficewalaSerpentCS — CHANGES_REQUESTED** [review]

File: hr_employee_firstname/models/hr_employee_public.py (hunk 1 above)
Comment (astirpe) [reply hr_employee_firstname/models/hr_employee_public.py:11]: On the public is set `readonly=True`. This is the way the employees V13 fields are made.

**Review by AmmarOfficewalaSerpentCS — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

**Pull request merged**